sensorprocess/testhelper.go

```diff
@@ -0,0 +1,269 @@
+package sensorprocess
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/golang/geo/r3"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/spatialmath"
+	"go.viam.com/test"
+
+	"github.com/viamrobotics/viam-cartographer/cartofacade"
+	s "github.com/viamrobotics/viam-cartographer/sensors"
+	"github.com/viamrobotics/viam-cartographer/sensors/inject"
+)
+
+type addLidarReadingArgs struct {
+	timeout        time.Duration
+	sensorName     string
+	currentReading s.TimedLidarReadingResponse
+}
+
+type addIMUReadingArgs struct {
+	timeout        time.Duration
+	sensorName     string
+	currentReading s.TimedIMUReadingResponse
+}
+
+var (
+	//nolint:dupword
```

Comment:
This was not triggered while it was in a different file. For some reason, the linter does not accept it in this file. Could not resolve this without a `//nolint:dupword`.